include/zephyr/net/wifi_mgmt.h

```diff
@@ -174,7 +174,11 @@ enum net_event_wifi_cmd {
 #define NET_EVENT_WIFI_DISCONNECT_COMPLETE			\
 	(_NET_WIFI_EVENT | NET_EVENT_WIFI_CMD_DISCONNECT_COMPLETE)
 
-/** Wi-Fi scan parameters */
+/**
+ * @brief Wi-Fi scan parameters structure
```

Comment:
properly terminate brief desc 
```suggestion
 * @brief Wi-Fi scan parameters structure.
```